src/features/farming/hud/components/Settings.tsx

```diff
@@ -56,7 +56,7 @@ export const Settings: React.FC<Props> = ({ isOpen, onClose }) => {
             className="absolute cursor-pointer z-20"
             onClick={onClose}
             style={{
-              top: `${PIXEL_SCALE * 6}px`,
+              bottom: `${PIXEL_SCALE * 26.4}px`,
```

Comment:
This may be causing the close button to be positioned strangely in the Settings Modal

<img width="350" alt="Screenshot" src="https://user-images.githubusercontent.com/41215134/205520060-40e134d3-86ba-4aa5-9e28-b68bbefcb435.png">

Reply:
<img width="353" alt="Screen Shot 2022-12-05 at 10 03 35 am" src="https://user-images.githubusercontent.com/17863697/205520946-00dd9c5f-1d13-434f-92c8-da276b54462a.png">